experimental/auth-logic/wrappers/wrappers.go

```diff
@@ -96,3 +97,12 @@ func EmitAuthLogicToFile(authLogic Statement, filepath string) error {
 	_, writeErr := f.WriteString(authLogic.String())
 	return writeErr
 }
+
+// SanitizeName takes a value that was parsed from something external to
+// authorizaiton logic and returns a new value that can be used as part of the
+// syntax for principal names and arguments in the authorization logic syntax.
+// At present, it removes hyphens. This is exported because it may be used in
+// the main package to get principal names.
+func SanitizeName(oldName string) string {
+	return strings.ReplaceAll(oldName, "-", "")
```

Comment:
For better readability, instead of removing `-`, perhaps replace it with some other permitted character (like `:` or `_`) as you suggested in #48.

Reply:
fixed

Reply:
fixed